src/js/components/FormField/FormField.js

```diff
@@ -474,14 +476,19 @@ const FormField = forwardRef(
                 event.persist();
                 if (onChange) onChange(event);
                 if (contextOnChange) {
-                  const debouncedFn = debounce(() => {
-                    contextOnChange(event);
-                    // A half second (500ms) debounce can be a helpful starting
-                    // point. You want to give the user time to fill out a
-                    // field, but capture their attention before they move on
-                    // past it. 2 second (2000ms) might be too long depending
-                    // on how fast people type, and 200ms would be an eye blink
-                  }, 500);
+                  const debouncedFn = debounce(
+                    () => {
+                      contextOnChange(event);
+                      // A half second (500ms) debounce can be a helpful
```

Comment:
Can we move this comment into `base.js`, since that is where the value is being set?

Reply:
Yup, done